packages/calcite-components/src/components/combobox/combobox.scss

```diff
@@ -250,6 +250,17 @@ calcite-chip {
   z-index: var(--calcite-z-index-sticky);
 }
 
+.no-matches {
+  @apply block px-4 py-2;
+  color: var(--calcite-close-icon-color, var(--calcite-color-text-3));
+  background: var(--calcite-combobox-background-color, var(--calcite-color-foreground-1));
+  cursor: pointer;
+
+  strong {
```

Comment:
We should be using `--calcite-font-weight-bold` for this.

Reply:
See [this thread](https://github.com/Esri/calcite-design-system/pull/12525#discussion_r2238056375) for a related discussion.